SampleUsages/Program.cs

```diff
@@ -19,43 +16,122 @@ public class Program : CmdHelper
     {
         public new static void Main(string[] args)
         {
+            FunctionLogs._logger = new ConsoleLogger();
             var p = new Program();
             ((CmdHelper) p).Main(args);
         }
 
+        [Command]
+        public void RunScenario(string scenarioFilePath)
+        {
+            var testScenarios = new List<TestScenario>();
+            using (StreamReader r = new StreamReader(scenarioFilePath))
+            {
+                string json = r.ReadToEnd();
+                try
+                {
+                    testScenarios = JsonConvert.DeserializeObject<List<TestScenario>>(json);
+                }
+                catch (JsonException ex)
+                {
+                    Console.WriteLine($"Failed to parse input file. {ex}");
```

Comment:
either exit the execution here or throw an unhandled exception with the "Failed to parse input file" message